src/metatensor/models/experimental/pet/train.py

```diff
@@ -137,7 +135,7 @@ def train(
         key: ModelOutput(
             quantity=value.quantity,
             unit=value.unit,
-            per_atom=False,
+            per_atom=value.per_atom,
```

Comment:
This should not be `value.per_atom` but `True`. `value.per_atom` indicate what's in the training set, while the value here indicate what PET can do. 

Can PET be trained on either per-atom values? Or only on per-structure energies?